src/test/java/showcase/neodymium/tests/pdf/PDFDownloadTest.java

```diff
@@ -0,0 +1,53 @@
+package showcase.neodymium.tests.pdf;
+
+import static com.codeborne.selenide.Selenide.sleep;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import com.codeborne.selenide.Selenide;
+
+import io.qameta.allure.Description;
+import io.qameta.allure.Owner;
+import io.qameta.allure.Severity;
+import io.qameta.allure.SeverityLevel;
+import io.qameta.allure.junit4.DisplayName;
+import io.qameta.allure.junit4.Tag;
+import showcase.neodymium.tests.AbstractTest;
+
+import org.apache.pdfbox.pdmodel.PDDocument;
+import org.apache.pdfbox.text.PDFTextStripper;
+
+/**
+ * @author kunze
+ */
+@Severity(SeverityLevel.TRIVIAL)
+@Owner("Georg Kunze")
+@Tag("smoke")
+@DisplayName("PDFDownloadTest")
+public class PDFDownloadTest extends AbstractTest {
```

Comment:
```suggestion
public class PDFDownloadTest extends AbstractTest
{
```

We normally place curly braces in the next line.